App.js

```diff
@@ -11,6 +11,7 @@ const MyButton = Object.assign({}, AmplifyTheme.button, { backgroundColor: '#65b
 const MyTheme = Object.assign({}, AmplifyTheme, { button: MyButton });
 
 Amplify.configure(config);
+Analytics.disable();
```

Comment:
I'm taking it this is what gets rid of the weird YellowBox we were getting?

Reply:
It got rid of the invalid promise error that we were filtering out